exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/TraceTranslatorTest.java

```diff
@@ -0,0 +1,198 @@
+            .build();
+    Map<String, AttributeValue> fixedAttributes = new LinkedHashMap<>();
+    fixedAttributes.put("fixed", AttributeValue.newBuilder().setStringValue
+        (TruncatableString.newBuilder().setValue("attributes").setTruncatedByteCount(0).build()).build());
+    fixedAttributes.put("another", AttributeValue.newBuilder().setStringValue
+        (TruncatableString.newBuilder().setValue("entry").setTruncatedByteCount(0).build()).build());
+
+    Span.Attributes.Builder attributesBuilder = Span.Attributes.newBuilder().setDroppedAttributesCount(0)
+        .putAttributeMap("myKey", AttributeValue.newBuilder().setStringValue
+            (TruncatableString.newBuilder().setValue("myValue").setTruncatedByteCount(0)).build())
+        .putAttributeMap("/http/host", AttributeValue.newBuilder().setStringValue
+            (TruncatableString.newBuilder().setValue(String.valueOf(3.14)).setTruncatedByteCount(0)).build())
+        .putAttributeMap("/http/status_code", AttributeValue.newBuilder().setBoolValue
+            (true).build())
+        .putAttributeMap("anotherKey", AttributeValue.newBuilder().setIntValue
+            (100).build())
+        .putAttributeMap("g.co/agent", AttributeValue.newBuilder().setStringValue
+            (TruncatableString.newBuilder().setValue("opentelemetry-java [0.6.0]").setTruncatedByteCount(0)).build());
+    for (Map.Entry<String, AttributeValue> entry : fixedAttributes.entrySet()) {
+      attributesBuilder.putAttributeMap(entry.getKey(), entry.getValue());
+    }
+
+    Span.Attributes allAtributes = attributesBuilder.build();
+
+    assertEquals(allAtributes, TraceTranslator.toAttributesProto(attributes, fixedAttributes));
+
+  }
+
+  @Test
+  public void toTimeEventsProtoTest(){
+    List<SpanData.Event> events = new ArrayList<>();
+    SpanData.Event eventOne = new SpanData.Event() {
+      //The SpanData.Event interfaces requires us to override these four methods
+      @Override
+      public long getEpochNanos() {
+        return 0;
+      }
+
+      @Override
+      public int getTotalAttributeCount() {
+        return 0;
+      }
+
+      @Override
+      public String getName() {
+        return "eventOne";
+      }
+
+      @Override
+      public Attributes getAttributes() {
+        return Attributes.newBuilder().setAttribute("key", "value").build();
+      }
+    };
+    events.add(eventOne);
+
+    Span.TimeEvents timeEvents = TraceTranslator.toTimeEventsProto(events);
+    assertEquals(1, timeEvents.getTimeEventCount());
+    //TODO: Figure out how to use timeEvents to call SpanData.Event methods to ensure those are working properly
+  }
+
+  @Test
+  public void toStatusProtoTest(){
+    io.opentelemetry.trace.Status myStatus = io.opentelemetry.trace.Status.OK.withDescription("Status description");
+    Status spanStatus = TraceTranslator.toStatusProto(myStatus);
+
+    //The int representation is 0 for canonical code "OK".
+    assertEquals(0, spanStatus.getCode());
```

Comment:
is there an enum you can replace 0 with for this? If not its fine

Reply:
I added a test in there with the enum "OK"